### ISSUE TITLE
Difficulty creating the Imperial Calendar from the Traveller RPG setting

### ISSUE THREAD

**Issue by Ermintrude-the-cow** [valentine195/obsidian-fantasy-calendar#30]
The Traveller RPG has a calendar whereby the first day of the year (Day 1) is called 'Holiday', then from Day 2 onwards it follows a seven-day week. There are no months, it's all a single bucket of days - 'Holiday' on Day 1, then from Day 2 to 365 it's a seven day cycle. (link to calendar here: https://wiki.travellerrpg.com/Imperial_Calendar). 

I either end up with the first weekday of the year being numbered '1' (it should be 2), or alternately day 2 falls on the second weekday of the year (it should be the first weekday). Would be great if I could find a way around this so that I could use your wonderful plugin

**Comment by valentine195** [comment]
So would the “month” display on this calendar just show all 364 days? Just trying to understand!

**Comment by Ermintrude-the-cow** [comment]
Yeah, there are no months at all, so the single 'month' is 364 days, numbered 002 to 365, with Day 001 being an intercalary day

**Comment by ckennedy666** [comment]
I sorta have this working by using an intercalary month of length 1 for Holiday.  Of course, the day numbers end up wrong.  Tried it with a Leap Day, and while that works on fantasy-calendar.com correctly with day numbers and everything, it fails in Obsidian Fantasy Calendar.  JSON for the Third Imperium Calendar available, if you can tell me how to send it in.

**Comment by Ermintrude-the-cow** [comment]
Thanks for that - yeah, the days numbers don't work out, which is a shame. Thanks for trying!

**Comment by Ermintrude-the-cow** [comment]
Oh, the other way to do it is to sat the first day of the week to Senday, but that leaves the weeks not lining up

**Comment by valentine195** [comment]
This is now possible by creating an intercalary leap day.
